tests/test_examples.py

```diff
@@ -31,8 +31,7 @@ def test_rat_examples(example_name):
     "example_name",
     [
         "DSPC_function_background",
-        # FIXME: https://github.com/RascalSoftware/python-RAT/issues/102
-        # "convert_rascal",
+        "convert_rascal",
```

Comment:
This test is splitting out at mat file (lipid_bilayer.mat) into the repo. Please make it so the file is written to a temp dir instead